web/src/components/Project.vue

```diff
@@ -23,26 +30,39 @@ import ProjectRepository from '@/repositories/ProjectRepository'
 export default {
   name: 'Project',
   props: {
-    project: String
+    project: String,
   },
   data() {
     return {
       logoURL: '',
       latestVersion: '',
       versions: [],
-      hideAvatar: false
+      hideAvatar: false,
+      isFavourite: false,
     }
   },
   async created() {
     document.title = this.project + " | docat"
     this.logoURL = ProjectRepository.getProjectLogoURL(this.project)
     this.versions = await ProjectRepository.getVersions(this.project)
     this.latestVersion = (this.versions.find((version) => version.name == 'latest') || this.versions[0]).name
+    this.isFavourite = ProjectRepository.isFavourite(this.project)
+  },
+  methods: {
+    switchFavourite() {
+      ProjectRepository.setFavourite(this.project, !this.isFavourite);
```

Comment:
I think we decided to not use semicolons (don't know if this was a good idea) but I would like to keep it consistant :)